examples/graph/app.js

```diff
@@ -0,0 +1,393 @@
+/* global window,document */
+import React, {Component} from 'react';
+import {render} from 'react-dom';
+import {json as requestJSON, csv as requestCSV} from 'd3-request';
+
+import DeckGLOverlay from './deckgl-overlay';
+import {default as GraphBasic} from './graph-layer/adaptor/graph-basic';
```

Comment:
while i find that pattern helpful for e.g. library code, in which the contents of the folder fronted by the `index.js` might grow over time, i think in example code it's less helpful -- there's no reason to assume we'll be adding more adaptors as time goes on, and it's one more step the dev following the example has to process to understand where the imports are coming from.  (also, it's a bit magical for those who don't understand that `import` resolution looks for an `index.js` first.)

agreed on the comment, will add.